includes/Admin/Settings_Page_Assets.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Handles asset enqueuing for the settings page.
+ *
+ * @package WordPress\AI
+ */
+
+namespace WordPress\AI\Admin;
+
+/**
+ * Manages scripts and styles for the admin settings page.
+ *
+ * @since 0.1.0
+ */
+class Settings_Page_Assets {
+	private const SCRIPT_FILE = 'build/index.js';
+
+	private const STYLE_FILE = 'build/style-index.css';
+
+	private const STYLE_FILE_PATH = __DIR__ . '/../../build/style-index.css';
```

Comment:
In places where we're traversing the path upwards like this, I wonder if we should use `WP_AI_DIR` instead?

```suggestion
	private const STYLE_FILE_PATH = WP_AI_DIR . '/build/style-index.css';
```

Reply:
resolved